bundler/helpers/v1/spec/native_spec_helper.rb

```diff
@@ -2,7 +2,6 @@
 
 require "rspec/its"
 require "webmock/rspec"
-require "debug"
```

Comment:
The `debug` gem was causing issues during dependency resolution. I removed it here, because we didn't appear to be using it anywhere.